eva-server/pom.xml

```diff
@@ -23,12 +23,20 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-        	<groupId>org.springframework.boot</groupId>
-        	<artifactId>spring-boot-starter-test</artifactId>
-        	<scope>test</scope>
-		</dependency>
-			
-	    <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-rest</artifactId>
```

Comment:
This dependency shouldn't be necessary: it allows to create repositories to query REST data but I don't think we are doing that?